### PR TITLE
Docs: add info for local development

### PR DESCRIPTION
**Pull request by anversa-pro**
## Issue ticket number and link
https://trello.com/c/bM8L1NCf

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [x] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

-

## What is the new behavior?

- Readme refactor and update

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by FelipeLosada1996 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,38 +31,66 @@ nvm install v18.16.0
...
+
+  | Key                  | Description                                    |
+  | :------------------- | :--------------------------------------------- |
+  | **PRIVATE_KEY_ID:**  | Project Id according Firebase (**mandatory**). |
+  | **PRIVATE_KEY:**     | Unique identifer for Project in Firebase       |
+  | **CLIENT_EMAIL:**    | Email created by Firebase for GService account |
+  | **CLIENT_ID:**       | Identifier for client connection               |
+  | **CLIENT_CERT_URL:** | Cert URL for Google identification             |
+
+  You can access the latest version of the required environment file at the following link: [Environment files](https://mckinsey.box.com/s/eqb5qsidkq0d4o8cochg49ndeoc094hp).
```
Comment (jose-sal-y-rosas) [inline README.md:48]: Please remove this line and only describe the variables regarding BOX.

File: README.md [hunk 2]
```diff
@@ -36,38 +31,66 @@ nvm install v18.16.0
...
+
+  Copy the `.env.example` file and rename it as `.env`. Then, configure the environment variables according to your specific environment:
+
+  | Key                  | Description                                    |
+  | :------------------- | :--------------------------------------------- |
+  | **PRIVATE_KEY_ID:**  | Project Id according Firebase (**mandatory**). |
+  | **PRIVATE_KEY:**     | Unique identifer for Project in Firebase       |
+  | **CLIENT_EMAIL:**    | Email created by Firebase for GService account |
+  | **CLIENT_ID:**       | Identifier for client connection               |
+  | **CLIENT_CERT_URL:** | Cert URL for Google identification             |
```
Comment (jose-sal-y-rosas) [inline README.md:46]: There are 31 environment variables, but there only 5 descriptions for these variable. Please can you add all of them?

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review comments

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**